packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx

```diff
@@ -69,7 +69,14 @@ const formatErrorMessages = (errors: FormErrors, parentKey: string, formatMessag
           )
         );
       } else {
-        messages.push(...formatErrorMessages(value, currentKey, formatMessage));
+        messages.push(
+          ...formatErrorMessages(
+            // @ts-expect-error TODO: check why value is not compatible with FormErrors
```

Comment:
@joshuaellis 

Providing 'initialerrors' to the Form seems like the best way to go. The only problem I'm having is with the FormError types here.

I've modified them to accept TranslationMessages but I can't work out why that has broken TS here in the CM. The type for values seems like it should be compatible.

(parameter) value: FormErrors<any> | TranslationMessage

any ideas?

Reply:
You could mark it as `todo`? it's probably just a bit of a bad recursive function 🤔